src/ralph/settings/base.py

```diff
@@ -168,3 +169,12 @@
     'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.LimitOffsetPagination',  # noqa
     'PAGE_SIZE': 10,
 }
+
+RQ_QUEUES = {
+    'ext': {
+        'HOST': 'localhost',
```

Comment:
use `os.environ.get('REDIS_HOST', 'localhost')` and make it working in docker too (in docker compose run redis container)